apps/src/lab2/types.ts

```diff
@@ -176,6 +176,7 @@ export interface LevelProperties {
   submittable?: boolean;
   finishUrl?: string;
   finishDialog?: string;
+  offerTts?: boolean;
```

Comment:
Do we have a way to toggle this in levelbuilder currently? Or if not could we log a quick ticket to add a checkbox for later?

Reply:
Not yet, since it seems unclear whether we'll go ahead with this functionality beyond some internal testing in the short term.  I've opened a [ticket](https://codedotorg.atlassian.net/browse/LABS-1061) to track adding a UI.